### ISSUE TITLE
vue/html-self-closin won't accept <input />

### ISSUE THREAD

**Issue by ananaszjoe** [vuejs/eslint-plugin-vue#974]
**Tell us about your environment**	
* **ESLint version: `6.5.1`**	
* **eslint-plugin-vue version: `5.2.3`**	
* **Node version: `8.12.0`**	

**Please show your full configuration:**
```json
module.exports = {
  root: true,
  env: {
    browser: true,
    node: true
  },
  parserOptions: {
    parser: 'babel-eslint'
  },
  extends: [
    '@nuxtjs',
    'plugin:nuxt/recommended'
  ],
  // add your custom rules here
  rules: {
    semi: "off"
  }
}

```

**What did you do?**	

```vue
<template>
    <div>
        <custom-component>
            <template v-slot="{ inputProps, inputListeners }">
                <div>
                    <input
                        v-model="inputValue"
                        type="text"
                        v-bind="inputProps"
                        :class="[
                            'autocomplete-input',
                            { 'autocomplete-input-no-results': noResults }
                        ]"
                        v-on="inputListeners"
                        @focus="handleFocus"
                        @blur="handleBlur"
                    />
                </div>
            </template>
        </custom-component>
    </div>
</template>
```

**What did you expect to happen?**
To lint my code with no warnings

**What actually happened?**
`$  warning  Disallow self-closing on HTML void elements (<input/>)  vue/html-self-closing`

**Comment by ananaszjoe** [comment]
If anyone else happens to forget that HTML `<input>` tags are end with `>` and not `/>`, now you know it too.
